scripts/main.js

```diff
@@ -31,17 +31,18 @@ $showNewAppFormButton.on('click', function () {
 $newAppForm.on('submit', function (event) {
   event.preventDefault()
 
-  // create app array for exisiting apps
   var app = {
     name: $newAppName.val()
   }
-  if (app.name) {
-    Apps.add(app)
 
-      .then(function (app) {
-        setRoute(app.id)
-      })
+  if (!app.name) {
```

Comment:
@gr2m Thank you so much for the code review :) Is it safer to check 'when app does not exist' than checking 'when app exists'? For example, we can prevent a user from creating an app without name ?